showcase/src/app/demo/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,4 @@
+<example-viewer [extensions]="['HTML', 'TS']" filesPath="breadcrumb/breadcrumb" exampleTitle="{{ 'SHOWCASE.DEMO.BREADCRUMB.TITLE' }}">
```

Comment:
Interpolation is not needed in this case. 
Could you please adapt this code like this: `exampleTitle="SHOWCASE.DEMO.BREADCRUMB.TITLE"`